doc/source/configuration/kayobe-automation.rst

```diff
@@ -0,0 +1,125 @@
+=================
+Kayobe Automation
+=================
+
+What is Kayobe Automation
+=========================
+
+`Kayobe automation <https://github.com/stackhpc/kayobe-automation/>`__ is a collection of scripts and tools that enable the automation of kayobe related operations through the use of CI/CD platforms such as those provided by GitHub and GitLab.
```

Comment:
For a first sentence, this is pretty intense. maybe something like:
```suggestion
`Kayobe automation <https://github.com/stackhpc/kayobe-automation/>`__ is a collection of scripts and tools that automate kayobe operations.
It is deployed and controlled by CI/CD platforms such as GitHub actions and GitLab pipelines.
```
My version of the content might not be 100% correct, I'm just suggesting the formatting style more than anything.